hazelcast/src/main/java/com/hazelcast/nio/NodeIOService.java

```diff
@@ -239,6 +239,11 @@ public int getConnectionMonitorMaxFaults() {
     }
 
     @Override
+    public int getHandlerMigrationIntervalSeconds() {
+        return node.groupProperties.HANDLER_MIGRATION_INTERVAL_SECONDS.getInteger();
```

Comment:
I don't know how often you are going to read this value; if it is just once I'm fine with it. Otherwise I suggest pulling it into a final field.